### PR TITLE
Enable EaaS in public staging

### PR DESCRIPTION
**Pull request by amisstea**

### CODE REVIEW

File: components/cluster-as-a-service/staging/external-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: cluster-as-a-service-hypershift-credentials
+  namespace: clusters
+spec:
+  dataFrom:
+  - extract:
+      key: staging/platform/terraform/it-cloud-aws-konflux-preprod-eaas01
```
Comment (amisstea) [inline components/cluster-as-a-service/staging/external-secrets.yaml:9]: @manish-jangra please review. I don't have access to this folder in vault so I can't verify some things like the key here or the ones referenced in `spec.target.template.data`.

File: components/cluster-as-a-service/staging/external-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - extract:
+      key: staging/eaas/konflux-eaas-stage
+  refreshInterval: 5m
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: appsre-stonesoup-vault
```
Comment (amisstea) [inline components/cluster-as-a-service/staging/external-secrets.yaml:15]: Should this be `appsre-vault` instead? We seem to have both in use.

File: components/cluster-as-a-service/staging/external-secrets.yaml (hunk 1 above)
Comment (manish-jangra) [reply components/cluster-as-a-service/staging/external-secrets.yaml:9]: vault path looks good

File: components/cluster-as-a-service/staging/external-secrets.yaml (hunk 2 above)
Comment (manish-jangra) [reply components/cluster-as-a-service/staging/external-secrets.yaml:15]: I don't think that makes any difference.

**Review by manish-jangra — APPROVED** [review]
looks good

**Pull request merged**